data/json/items/armor/jewelry.json

```diff
@@ -1293,6 +1300,14 @@
         "color": "yellow",
         "weight": 5
       },
+      {
+        "id": "saint_necklace",
+        "expand_snippets": true,
+        "name": { "str": "Saint Medallion" },
+        "description": "A necklace made of a fine gold chain with a medallion bearing the image of a Catholic Saint.  The back reads \"<catholic_saints> pray for us.\".",
```

Comment:
This feels awkward if <catholic_saint> expands into a description in addition to a name. That scheme also does not work for translation.

Reply:
Move the entirety of the quote on the back to the snippet.